components/segmenter/src/line.rs

```diff
@@ -19,97 +19,111 @@ const UNKNOWN: u8 = 0;
 #[allow(dead_code)]
 const AI: u8 = 1;
 #[allow(dead_code)]
-const AL: u8 = 2;
+const AK: u8 = 2;
```

Comment:
This probably breaks compatibility with current data structs, so you need to bump the version.

Reply:
I have already bumped to V2 for 2.0. Should I bump to V3?

Reply:
V2 isn't released yet so it's fine as long as long as this only affects V2 structs (looks like dictionary and LSTM are still on V1)